tests/setup.py

```diff
@@ -13,7 +13,8 @@
     package_data={'': ['*.md']},
     include_package_data=True,
     install_requires=[
-        'ducktape==0.8.0', 'prometheus-client==0.9.0', 'pyyaml==5.3.1'
+        'ducktape==0.8.0', 'prometheus-client==0.9.0', 'pyyaml==5.3.1',
+        'kafka-python==2.0.2'
```

Comment:
is this yapf formatted?

Reply:
yeah 🤷 . I think if I put a comma at the end it formats it better. I'll change it..